src/Tizen.NUI.Components/Controls/AlertDialog.cs

```diff
@@ -175,425 +116,369 @@ public View TitleContent
+            };
+        }
 
-            if (type == DisposeTypes.Explicit)
+        private TextLabel CreateDefaultContent()
+        {
+            //FIXME: Needs to separate GUI implementation codes to style cs file.
+            return new TextLabel()
             {
-                if (popupTitle != null)
-                {
-                    Utility.Dispose(popupTitle);
-                }
-
-                if (popupContent != null)
-                {
-                    Utility.Dispose(popupContent);
-                }
+                HorizontalAlignment = HorizontalAlignment.Center,
+                VerticalAlignment = VerticalAlignment.Center,
+                BackgroundColor = new Color(1.0f, 1.0f, 1.0f, 1.0f),
+                Size = new Size(360, 200),
+            };
+        }
 
-                if (popupAction != null)
+        private View CreateDefaultActionContent()
+        {
+            //FIXME: Needs to separate GUI implementation codes to style cs file.
+            return new Control()
+            {
+                BackgroundColor = new Color(1.0f, 1.0f, 1.0f, 1.0f),
+                Size = new Size(360, 80),
+                Layout = new LinearLayout()
                 {
-                    if (positiveButton != null)
-                    {
-                        Utility.Dispose(positiveButton);
-                    }
-
-                    if (negativeButton != null)
-                    {
-                        Utility.Dispose(negativeButton);
-                    }
-
-                    Utility.Dispose(popupAction);
-                }
-            }
+                    LinearOrientation = LinearLayout.Orientation.Horizontal,
+                    LinearAlignment = LinearLayout.Alignment.CenterHorizontal,
+                    CellPadding = new Size2D(10, 0),
+                },
+            };
+        }
 
-            base.Dispose(type);
+        private void OnRelayout(object sender, EventArgs e)
+        {
+            //FIXME: Needs to separate GUI implementation codes to style cs file.
+            CalculatePosition();
         }
 
-        /// <summary>
-        /// AccessibilityGetName.
-        /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        protected override string AccessibilityGetName()
+        private void CalculatePosition()
         {
-            if (!String.IsNullOrEmpty(Title))
+            var size = Size2D;
+            var parent = GetParent();
+            Size2D parentSize;
+
+            if ((parent != null) && (parent is View))
             {
-                return Title;
+                parentSize = ((View)parent).Size;
             }
             else
             {
-                return Message;
+                parentSize = NUIApplication.GetDefaultWindow().Size;
             }
+
+            Position2D = new Position2D((parentSize.Width - size.Width) / 2, (parentSize.Height - size.Height) / 2);
         }
     }
-}
+}
```

Comment:
could you add new line at the end of file?

reference: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
Thank you for your comment.
The code has been updated.